__tests__/app.test.js

```diff
@@ -97,6 +97,76 @@ describe('App Tests', () => {
                 })
             })
         })
+        describe('Added Feature: GET `/api/articles` test', () => {
+            test('topic query: filters the articles by topic value specified in the query.', () => {
+                return request(app)
+                .get('/api/articles?topic=mitch')
+                .then(({body: {articles}}) => {
+                    expect(articles.length).not.toBe(0);
+                    articles.forEach(article => {
+                        expect(article.topic).toBe('mitch');
+                    })
+                })
+            })
+            test('topic query: filters the articles by topic value specified in the query and is not case-sensitive.', () => {
+                return request(app)
+                .get('/api/articles?topic=MiTcH')
+                .then(({body: {articles}}) => {
+                    expect(articles.length).not.toBe(0);
+                    articles.forEach(article => {                        
+                        expect(article.topic).toMatch(/MiTcH/i);
+                    })
+                })
+            })
+            test('sort_by query: sorts the articles by any valid column(defaults to created_at)', () => {
+                return request(app)
+                .get('/api/articles?sort_by=title')
+                .then(({body: {articles}})=> {
+                    expect(articles.length).not.toBe(0);
```

Comment:
Could be more precise here by asserting for the specific length you expect it to be, remember that the test data should always remain constant so there's no reason to expect the length to ever change